jobs/fxci-taskcluster-export/fxci_etl/loaders/bigquery.py

```diff
@@ -0,0 +1,67 @@
+from abc import ABC, abstractmethod
+import base64
+from collections import defaultdict
+from dataclasses import asdict, dataclass
+import json
+import os
+from pprint import pprint
+from typing import Any
+
+import dacite
+from google.cloud.bigquery import Client, Table
+
+from fxci_etl.config import Config
+
+
+@dataclass
+class Record(ABC):
+    @classmethod
+    def from_dict(cls, data: dict[str, Any]) -> "Record":
+        return dacite.from_dict(data_class=cls, data=data)
+
+    @classmethod
+    @abstractmethod
+    def table_name(cls) -> str: ...
+
+    @abstractmethod
+    def __str__(self) -> str: ...
+
+
+class BigQueryLoader:
+    def __init__(self, config: Config):
+        self.config = config
+        self._tables = {}
+
+        self.client = Client.from_service_account_info(
+            json.loads(
+                base64.b64decode(config.bigquery.credentials).decode(
+                    "utf8"
+                )
+            )
+        )
```

Comment:
I think we should rely on Airflow SA for write access to BQ, so this would become:
```suggestion
        self.client = Client()
```

Reply:
Sounds good, I'll make the change. This is currently running in an EC2 instance in our `moz-fx-dev-releng` project, thus why I was using custom credentials.